packages/chart/src/hooks/useTooltip.tsx

```diff
@@ -96,10 +96,10 @@ export const useTooltip = props => {
 
     const closestXScaleValue = getXValueFromCoordinate(x - Number(config.yAxis.size || 0))
 
-    const includedSeries = visualizationType !== 'Pie' ? config.series.filter(series => series.tooltip === true).map(item => item.dataKey) : config.series.map(item => item.dataKey)
+    const includedSeries = visualizationType !== 'Pie' ? config.runtime.series.filter(series => series.tooltip === true).map(item => item.dataKey) : config.runtime.series.map(item => item.dataKey)
```

Comment:
I'm getting an error on dev-8719. note that runtime is not saved to the config so the logic should be able to handle and undefined scenario: https://github.com/CDCgov/cdc-open-viz/blob/495f22439e5c25e05de2ab6c4f44597cd3efce17/packages/core/helpers/formatConfigBeforeSave.ts#L87

Reply:
Can you send the config that is breaking? In the chart component, the updateConfig function initializes runtime which is called on first load. We had to fix it for standalone DataTables because they may not have a runtime, but charts should always have one